packages/esm-patient-chart-app/src/ui-components/patient-chart.component.tsx

```diff
@@ -19,13 +19,13 @@ interface PatientChartParams {
 const PatientChart: React.FC<RouteComponentProps<PatientChartParams>> = ({ match }) => {
   const { patientUuid, view, subview } = match.params;
   const [loading, patient] = useCurrentPatient(patientUuid);
-  const sessionUser = useSessionUser();
+  const sessionUser = null;
```

Comment:
That seems like a leftover from testing (it should still be `useSessionUser`, not always `null`).

Reply:
Oh yes, so sorry about that. I'll push the changed asap